internal/topology/variables/doc.go

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package variables implements validation and defaulting for ClusterClass variables.
```

Comment:
Q about the package location, should this go under internal/topology? (I'm assuming it should be used both by webhooks and the topology controllers)

Reply:
Just to confirm. Do you suggest to move the package from `internal/topology/variables` to `internal/topologies`?